api/src/application/user/user-controller.js

```diff
@@ -157,13 +155,13 @@ module.exports = (
           let title = 'Email Verified';
           const profile = user.get('profile');
           profile.email_verified = true;
-          await userService.update(user.get('id'), {
-            email: bookshelf.knex.raw('pending_email'),
-            email_lower: bookshelf.knex.raw('pending_email_lower'),
-            pending_email: null,
-            pending_email_lower: null,
-            profile
-          });
+          user.set('email', user.get('pending_email'));
+          user.set('email_lower', user.get('pending_email_lower'));
+          user.set('pending_email', null);
+          user.set('pending_email_lower', null);
+          user.save();
```

Comment:
Should we `await` the `save()` to confirm it actually succeeds before sending the webhook?

Reply:
mmmmmmmmm yes